tasks/configure_gitlab_ce.yml

```diff
@@ -155,20 +155,46 @@
     mode: '0644'
   notify: [ 'Restart gitlab' ]
 
-- name: Install GitLab service script
+- name: Install GitLab SysVinit script
   shell: cp -f {{ gitlab_ce_git_checkout + "/lib/support/init.d/gitlab" }} /etc/init.d/gitlab
   register: gitlab__register_init_script
-  when: gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed
+  when: ((gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed) and
+         (gitlab_use_systemd is defined and not gitlab_use_systemd))
+
+- name: Install GitLab systemd service files
+  template:
+    src: 'etc/systemd/system/{{ item }}.j2'
+    dest: '/etc/systemd/system/{{ item }}'
+  with_items:
+    - 'gitlab-mailroom.service'
+    - 'gitlab-sidekiq.service'
+    - 'gitlab-unicorn.service'
+    - 'gitlab-workhorse.service'
+  register: gitlab__register_init_script
+  when: (( gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed) and
+         (gitlab_use_systemd is defined and gitlab_use_systemd))
 
 - name: Reload systemd daemons
   command: systemctl daemon-reload
   when: (ansible_service_mgr == "systemd" and gitlab__register_init_script|changed)
 
-- name: Enable GitLab service
+- name: Enable GitLab SysVinit service
   service:
     name: 'gitlab'
     enabled: True
-  when: gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed
+  when: ((gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed) and
+         (gitlab_use_systemd is defined and not gitlab_use_systemd))
+
+- name: Enable GitLab systemd services
+  service:
+    name: '{{ item }}'
+    enabled: True
+  with_items:
+    - 'gitlab-sidekiq.service'
+    - 'gitlab-unicorn.service'
+    - 'gitlab-workhorse.service'
```

Comment:
After a host reboot the processes show up normally in `systemctl status gitlab-mailroom` so maybe that's just an initialization problem. Still, this service should probably be started automatically.